### PR TITLE
Feat: Statistics rewards

### LINKED ISSUE #19: Statistics
I just realized that if we remove the assigned tasks, then it messes with the stats. How can we preserve the stats for marketing purposes? It would be great to write a JSON object with the following:
```typescript
type Statistics = {
  rewards: {
    notAssigned: number;
    assigned: number;
    completed: number;
    total: number;
  },
  tasks: {
    notAssigned: number;
    assigned: number;
    completed: number;
    total: number;
  }
}
```

Perhaps this feature needs to be reimplemented? Its already in use on ubq.fi and I have ideas for how to use it on Twitter too.

_Originally posted by @pavlovcik in https://github.com/ubiquity/devpool-directory/pull/1102#discussion_r1516366949_

### PR DESCRIPTION
**Pull request by devpanther**
Resolves https://github.com/ubiquity/devpool-directory-bounties/issues/19

- Changed total-rewards.txt to total-rewards.json

Test Run: https://github.com/devpanther/devpool-directory/blob/development/total-rewards.json

### CODE REVIEW

File: helpers/github.ts [hunk 1]
```diff
@@ -251,28 +252,74 @@ export async function getProjectUrls() {
...
       if (priceLabel) {
+        // ignore pricing not set
+        if (priceLabel.name === "Pricing: not set") return;
+
         const price = parseInt((priceLabel.name as string).split(":")[1].trim(), 10);
-        totalRewards += price;
+
+        // Increment rewards statistics, if it is assigned but not completed
+        if (isAssigned && !isCompleted) {
+          rewards.assigned += price;
```
Comment (0x4007) [inline helpers/github.ts]: Does this have proper error handling for NaN etc?

File: helpers/github.ts [hunk 2]
```diff
@@ -251,28 +252,74 @@ export async function getProjectUrls() {
...
+        } else if (!isAssigned && !isCompleted) {
+          rewards.notAssigned += price;
+        }
+
+        //Increment completed rewards statistics
```
Comment (0x4007) [inline helpers/github.ts]:
```suggestion
        // Increment completed rewards statistics
```

**Review by 0x4007 — APPROVED** [review]
Code looks good. Would like to see a successful test run, and its results.

**Comment by devpanther** [comment]
here @pavlovcik 

https://github.com/devpanther/devpool-directory/blob/development/total-rewards.json

File: helpers/github.ts (hunk 1 above)
Comment (devpanther) [reply helpers/github.ts]: I've added an error handling now, although i had a check for price not set earlier but NaN check would make it better

**Review by 0x4007 — APPROVED** [review]

**Pull request merged**